src/Tribe/Service_Providers/Context.php

```diff
@@ -294,6 +313,11 @@ static function ( $post_type ) {
 				],
 				'organizer_post_type'  => [
 					'read' => [
+						Tribe__Context::FUNC          => [
+							static function () {
+								return ! empty( tribe_get_request_var( Organizer::POSTTYPE, false ) );
+							}
```

Comment:
* :no_entry_sign: Each array item in a multi-line array declaration must end in a comma
* :no_entry_sign: Comma required after last value in array declaration

<sub>hash: f2076fbff048be996241c23ed9034a20</sub>